pkg/ovsdb/transact.go

```diff
@@ -1043,9 +1044,14 @@ func doInsert(txn *Transaction, ovsOp *libovsdb.Operation, ovsResult *libovsdb.O
 		}
 	}
 
+	klog.V(6).Infof("validate schema of DB %s on table %s", txn.request.DBName, ovsOp.Table)
 	ok, err := txn.schemas.Validate(txn.request.DBName, ovsOp.Table, &ovsOp.Row)
-	if !ok || err != nil {
-		klog.Errorf("Failed to validate table %s/%s", txn.request.DBName, ovsOp.Table)
+	if err != nil {
+		klog.Errorf("table %s validation returned %v", ovsOp.Table, err.Error())
```

Comment:
the error should be rather be:
```
klog.Errorf("Failed to validate table %s/%s: %s", txn.request.DBName, ovsOp.Table, err.Error())
```

as it now:
1. says that the validation failed (for example I could just be missing the schema while value is actually valid)
2. you are missing the database context <dbname>/<table>